lib/mdx.mli

```diff
@@ -69,7 +69,7 @@ val parse_lexbuf: syntax -> Lexing.lexbuf -> t
 
 (** {2 Evaluation} *)
 
-val run: ?syntax:syntax -> f:(string -> t -> string) -> string -> unit
+val run: ?syntax:syntax -> force_output:bool -> f:(string -> t -> string) -> string -> unit
```

Comment:
you can even use an optional argument (with default `false`) and update the doc strings :-)

Reply:
Done again, my bad for the docstring :p